pkg/menu/menu.go

```diff
@@ -37,6 +37,18 @@ func max(a, b int) int {
 	return b
 }
 
+func Init() error {
+	return ui.Init()
+}
+
+func Clear() {
```

Comment:
I don't see this being used anywhere?

Reply:
I'll take that out. I think the menus already clear themselves with `ui.Clear()`.